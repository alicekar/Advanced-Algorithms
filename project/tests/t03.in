512
11000001000100111011010001101001
01011010110011010101110010111001
01100100010000001110001000011011
00111001100100111111100111111011
10101011111101100100001001100010
01001001101011011111100110000001
10110110000111101111111110111110
00000010011111100111000001111001
10011111101000000110100110010111
10100011110111101110001011110111
10001001100010000011100000001011
00101011010101010110111000011111
11010000100011010001011001000110
01100110100010000000111001010000
01110011011101101100111111101001
10010011001100101110000001100110
10111000000110100101100101001010
00010000000010101111101100110011
11001010011000000100000100100010
00111101010011010001011110011101
01101000101101011101001111010101
10110111001011111001110111101011
10010001110101101110110011100010
11011111100000000011110110010101
10010011010001000000111011000111
01111111111101110011011001010010
00000101111111001010010110111100
11011111010101100000001111100011
11000100100110101001010010111011
10101010110111000111000011000100
00011100011011111011010110001101
00011110010000011101001100011011
01000000000111100001110010111110
01111000100111111111001100101011
10100011100100010101011010000010
00001010100001011000001110001111
00101001010111100110101111001001
10000101010011111110101111000111
01100000010100111110011111001001
11111010011110001011010011111110
10000010101011110111000010100000
01100001111111000101111111100110
01110000010001010010101100001000
10100100101000001110101011110001
11011000010101000110011001011101
01001000101100100010111010111010
01010110000100010110001000010010
01100100110100000101001010110001
00110110011011001011001101111010
10001101010110101101101010101100
10010110110000010100000110100100
11010001011011110110100111100000
00000101101111111110001101011101
01000111100010111001110101011010
00110101110111111100101001110110
10111111000100000101101011111011
11100010100101010001010010000010
11001100111100110011011010110011
11001010111011000110010101010010
11101011000001100001001110010100
11010110110000111100001001111000
10000111011010010010110001111010
01010101111011010101010111001010
00100001000100100101111001101001
00110111000101111001011011001100
11101111011011010100101110000011
00010111000011001001101000111110
01000010100000100110110001001101
01100111101110110100100101001111
11100000111110001110100101000000
10110010001110000001000100011110
00100001010100010001000111010100
11100100010011100100110011111011
11010011000100101001000111001000
01101101111000101101001111101010
01111010000100001001110011100001
01000001101000000101101000010111
10111010111110101101000010110101
11011100010101110000011000110011
00011101101101100001001100110000
01110100000100100100111001011111
01010111011011011000100101110100
11110011100010010101011100101110
00111101000011100111000110010110
11100100111001111000000111000001
01100001011010110001101111000111
00100111101001000000110101001110
01000100110110000001000100111010
10111101010111101100001111100001
11000111111111100100001010101101
11101011010101101100111001101010
11000011000000011001101001000110
10010011110101110001110111000110
10101111111000001111101011001011
10100001010110100000000010011100
11111110101111101011100101010011
10001110100001000000100110000000
00001100101100101101111010001000
01001111111001001001011011101010
11110110001101001111100111100000
10010001101101001110001101010110
11111100001000001110001101011000
01001011011111001000110011000100
10001111011001111000100100101111
01110101110011000110110000010001
00111110100001011100001110111010
00000100101011000111100001110011
00010100110101111010000000000000
00101011101000111001001110010000
10100001001001011001100001101011
10101011000010110010000111100001
10111110111100011011100001001111
11111110110000010010001010001001
10110101101111000100100100111010
01001110010110100010101110011010
10010001010100110001100010100101
01110111000110111010100011110111
10101110010011011010111010110000
11000110000100101101101101001011
10111101011110010010010101001001
00111001011011011001111000001010
00100111111011011011100100101011
01110000111000001000111110100001
01110110101011101010111001110010
11010101000010010001110110010000
01001010101110111110010001010110
11111010010100000110111100000001
10010010010010010001000000110000
10111011011111001010000001011010
01100011101001011111011000001011
00011010111011001110101011000010
00101001110010111000110101111101
01011101111000001101101001011100
01111010001111110101011010110010
01110100100001011111111011100100
10001100001100000101011101011011
00000010111000110001011101010111
11100101110000001111111101010011
01100010001001110111011000111011
01010111000100010000011011000011
01001110110000011001100110111001
01001011000011000010011111000010
10100001100110110001010100010110
01101010011000010100000000011111
11001000101111110111101000100011
01000101010100111011000111000001
01110111001010011010011010110101
10101111001010110010010100111101
00000101101001110101010101101010
00000010001010010001011110111111
11100110001100100010000001010100
11001010000001111100110100000101
01110110000000110010111110100101
00000001101001110010001001011100
01011101000110101001001010111000
00110011110001110100100100111001
01010111010000010011100001101010
00110101111100000011011110110010
00110110011111011010111100010111
01001010110000100010101110111111
00001000110011001110111010101000
01000011100101101111100001010110
00110001010011000100101011011001
01111101001101101100000101110101
11000110110000011000100110001110
01101100111110100100110010111101
11100001010001101011011101000001
00011110010010010010010110110100
01011010010011010010101100000111
10101010110010011000011000100000
11100111101010110011011011100101
11110011100000101011000000000111
10111000011000101111111000001010
10000011110011001001001000010010
10000011101101010101111010110110
11010001100011100011000101011000
00111111011001000010100101111001
01010001010001010000010001110110
00010110110101111001011110110011
01111001101010100001000110000111
11101100101110111101011000110010
00111100111101110111001110010111
01111110111011110111001010011000
01011100010000101011100101001010
11110111100001011010000111000010
00001111010010010100111010000111
11110011110001011110011100010010
01100001110010111000111110111111
00111011111111011001000100011110
11111101011111110001110000001010
10110001000101110111001011011011
01001000011101110101000100111010
01110101001001010110100001100111
10000110111100000010000111101101
00110111100100101100010101100000
01100001110100101010000101101011
11000011001001101011100011100111
11001010010111111000011110000100
11111011001011111001011110000000
01010001100011100001101000001110
01000000100100101101111101101111
11111110101110000001010100110011
01011100011110011100000010001010
10010010111000100110101000110100
01110111011101000110100110010100
01011110111011000110111101111111
01101000100111000000111000001001
01001101001101001100001000110101
00011000001101000000000001000010
01100101110110110001111010101000
10110101011111111101011100001110
00101111111000000100011100010111
00000100110001000101100010000011
00000110000010010011101100111111
10101101110110010100011001101010
01101101001000010001011111001000
00011000100001001011000001100010
01110010101001101100110110000111
11011100000110001011110110011011
10001100001010101111110110111111
11001010110111011011100110111011
10001001000111011010001000100000
10111101000110001101110101100000
11111111110000100110110110001001
11011111000110000011101101011101
01011111011110101111101001111011
11100100000010100110101111011000
00111010001100101111110101001001
00011000100110101111110010000100
01111110111011111010010000000101
01011111000011000000100100101001
11111001101100110100010101101011
10001110001011001110100110011001
11010110000001101000111101100110
00101110100101101101001001101000
11011110111001100011111101111000
10100011110010111110101101100011
10111000011001111100010110101110
00111101001010010001111111100001
11100010001001001101011111110111
01110110010011110110100111000000
11001000000101000010111010100110
11000000100111111100000000001011
11111010001010000001011001101110
01100011000001110101101000011011
11100000111110001011100100000001
11101000100100101001101001001100
00100001101101010011111000111000
01100101011101101000111101110001
11000001100101000001110110101101
10101101110010000110000000110010
01011011010000111101101000001001
10101100111110011101011011010110
00010001111000100001011111101000
00101101011110011101110001110111
00010111010101000010000111001011
00111001010011100101000101001010
01111110011101010111001010011000
10100111101010011110101100000011
10010011110000111100010011001100
01111001101011011100010011010100
11010100011110000101011111001000
00111100000001110011010100011011
11000110101111101100100001111010
00011010100101111000111111000011
01100110100100101111011011011111
11010100011011111110101010100111
11101011101011010010100100111011
10010101000000111111100101010010
11101110001101101111110010011001
00010001001110011100000100001100
10111111001100011011110110111010
00011111011011101000011011111110
01110101001100101111011010101001
01110100010110001111011100111001
00111011000011101010001000000101
00001100100010001001011000111100
01100100110010100001000110111011
01010100011011000011100011001000
00100101011101110010000111010011
11000010110100101010000001111110
01010111001111110111111100001011
00000011001111011000001011110111
10010000011000000011010011010000
10100111011000011101101000110011
10011001100011111111110100011001
00001001110000110100101111011000
00101101000110011001000111001101
10110011110110000100100110000110
00011100101110000111111100011011
01010100001011101010011101010111
01001010101101100000111100000110
01001011011001110001010100101111
01001001100001100011111100110001
00100111110111101100111000100100
11111000101000011011101011010110
10000110111110101100110111001001
01011011011101010010111111111111
01010000010010111001110001100010
00000000000100110001101000010101
11101001000101101011101010011101
10010110110011001001110100100001
11110100001011111010101011011001
01011000100010110110111100011010
00000101100111110010101010100000
10010110001011011011011000011100
00000101010000110010100000000011
01000110000010110010011110000011
10011001001000111001110010110000
00011111100111001011101110101000
00100101001000110010010000101111
11000010100111110001010111001000
01000011111101111010100111111011
11110101101111100111011101011100
01000011001100100011010010001001
11110100111000000100110011101111
01010111000101000001001000101011
00000100101011010111011110011110
01010101100100110001010011011011
00000111000111010010100110010111
01100101010111111100000011010001
11110000001001101001100111011001
00100110110010110001100001101011
01111101011100110111000010111101
01000110001011100100000011111100
00011000001000100101111100010111
11010101010110110100011100100101
00110100001001110011110100111100
11111011000100101101101000111110
01011100001010001010011010101100
10000101011101011000111111010100
10011110110001000000111110011010
11100010001101100100000110011011
01101100100001111101100000110000
01110001111110101100011111000001
10001010110010100111101011100101
01000000011111000100110111101110
01010011110100110101111010111111
01010111111011100110110010011000
00111000110011010010000100101111
11111100100111110001101111010110
10000111101101111101111111001010
11000110001000011001100001100100
00011000011110110110011000000010
10011000101011101100011110101111
11100011010011100010000110100111
11001000111010010110100101101011
00011101100011000011010110101001
01110011001010000110000100010000
01111011110011010000101011010110
11000010000000100011001011110000
01101011100101111011100010001000
11001100010101100101000011101010
00100000110100100101110010000111
01001100000010001100010110110111
01010101000011011011010111111110
11110100110011010110011110001011
00001001111101000011101111100110
00001000111111101111011110101101
11011011001110111011010110101101
11010011011101101111010111111110
10010000101010111100011110011011
11011011000001000111111000001001
00001000011000000011011100001011
00001110010011001111110101101101
10001010111001100010111101001000
11000001100010111001011111100010
01100010100001000001111110101111
00100011001101000111000111001100
00010111011000001101110001111111
01000110011110011010111101000000
10101011110111100000000011000000
00101011011101100010011111101110
01100100110100110001100101000011
00110000011001001001010010100100
01100111101001111101000101100110
10011010011010011101001110010000
10000111001110010110000000011111
00001010110011011101010101110111
11010100010111000110101100001010
11111010110111111100000110011001
11110001100101111110100101000111
01010001110100100000001110001010
11010010011100000110110001000111
10011011110011100010011100001111
10001110000101010000110011111000
00100001001111011000110101001000
01001110011101010011111000100100
00010010010101010001001000001010
01100010111000000011111001000111
10110101101000000101011011101011
01100010011110010101011011100010
11110111101111110001101100110111
11011010111011111111010010101100
00010111011100000001111001100001
01010001000100110111101100111111
01000100010111111101100010110111
10111001110111001110110010111101
11100010001111110100110001010010
00011100111001000101100111101101
00011011100010101000100001000111
10011010100011011000110000011000
01010000101010011001000101101111
00001110010101000001000011010011
00000101100011110010010010100001
00011111100000011101100001110011
01001000011011100110111110111011
01001111011010101110111100110101
01001011101010000010111110110001
01001110101010100000111011000011
11101000000011001111101110011111
00111110011110011101110001011110
01100100100111000101011111000000
10011000011111000100001100111010
11110001000001000111000100000110
10001000110000101000101000001100
11011100110100010111101101110000
10011101000010100101011111000001
01000101100101101100101011111011
10000000011101001101000011011111
10000011101100011110101001101001
00000110010100001010101010111110
00111111100101001011100001000100
01010010000001111001101111111010
00110000000000111001000001001011
01110011101101001010110111111111
00000111101001100111000101110001
01000110010100100011110100000100
11000010111010000111100111010010
10011111011111100111101011011111
00111111001101100100111010001010
00110101001100110001000110010010
00111101001110001010001010110101
11001000000111011010110110101100
10101111100001001000011000101110
10011101111011101000001001001000
10111110111010010110110111101111
00110001101010011010000010101011
01000101001000111001111001001101
10010110000000100111010100010101
11010000010100110100000000000110
01111010100101111101001010111000
00001001011000001001110111001100
11101011001010100111100100000001
01101110111011001111011010100110
11011100110010001110010011011111
01111110101100000011101010101100
10000001001000100111000110110010
10111011001000110000010011000101
00000000110000001101000011011001
00100001000100101001011010110100
10101100101011010010001000011111
10000111011000000001011101011001
10100101100000110101010111001100
11110010000100110100100101101100
11000010111100111100011000011111
00011011001011100000110111010000
00110010100001010111111010110000
00000101000001110000010100011010
11011100100000001110101101111001
10000111101110010111001000111111
01100010110001101100011001011001
10100101111101111111011101111010
10101001100000111010010001010110
01100000000110110110000010000100
11100110001100001011001100010101
00010001100010001000101111100000
10101011110111100010101011110101
00110001111011010011111110010100
00001001001110000011001000001001
01101110111110100100111000011001
10011111111111100101100100111110
00110010111100011001111001101010
00011111100101011001111010000111
00001010101110111100101100100100
10010011101101000010011011010001
01011001110101100100001010111110
00100100100111111011110001000011
01001011000110010110111111110011
10001101010101011010100111101011
01001011100111101100101100001010
10010100110011100110010100111101
10101001101011010001111010100100
10110010001001000001000011110000
10111100000001100001110001101001
11011111010110011001011101110010
11011111011100011100010110100110
01111010101110101111110110001111
11101000001000110111111100001000
01011101011100000000001010111010
10010100100111000110100011010011
11001100010010100000110010111100
01111001100111011000101001011110
10111111010101111111010100100110
00000000001111111010110110011100
10001110100010111011110010011101
00010001010100010100101001001011
11100010111010111110000100101111
11000011110101110101111101010111
11011010010100101101010001100001
11001001111100111100100001011000
00000110110010101101010110101011
01110001101001111001110110110101
01001000100001111001111111000100
01010110001000100000111100110101
11111010000001110000010100011101
11011010010111100100010110010100
00101001000100111101101110000101
11101011000100111101110100110001
00111001111111100110010111100111
01111001100010010100000100011010
10011110011010110101100101011011
5
