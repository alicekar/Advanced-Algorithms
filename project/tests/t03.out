10 468
35 57
70 246
88 278
166 471
183 258
305 405
378 461
